qtpy/QtGui.py

```diff
@@ -1,17 +1,18 @@
-#
+# -----------------------------------------------------------------------------
```

Comment:
After the merge with master to resolve the conflict I think the replacement of `PythonQtError('No Qt bindings could be found')` with `QtBindingsNotFoundError()` got lost for this module?

Reply:
Yeah, my bad. I tried to resolve the conflict here inside Github instead of fetching and rebasing in vscode like I should have. Let me undo this real quick and fix it the right way.